sdk/src/sealance/merkletree-support.ts

```diff
@@ -0,0 +1,237 @@
+import { Field, Plaintext, Poseidon4 } from "../wasm.js";
```

Comment:
Can you just rename this file to `merkle-tree.ts` and make the directory Sealance is in an integrations folder?

The path should then be `integrations/sealance/merkle-tree.ts`